### PR TITLE
Fix nested if statement flow problem in debugger

### LINKED ISSUE #13040: [Debugger] Incorrect line flow when debugging nested if statements
**Description:**
When debugging a code that contains nested if statements, an incorrect line flow occurs.

**Steps to reproduce:**
Check the below image:

debugger

After line number 5 when executing STEP_OVER command, it goes to line number 8, which is unexpected and incorrect. 

**Affected Versions:**
0.990.2

**Suggested Labels (optional):**
 Component/Debugger, Type/Bug, Severity/Major

**Suggested Assignees (optional):**
@pranavan15

### PR DESCRIPTION
**Pull request by pranavan15**
## Purpose
This PR fixes #13040. The root cause for the aforementioned issue is `GOTO` instructions got assigned the line number of the last statement of the `if block` it belongs. Hence, the debugger shows the last line of the inner if block as a line hit. This issue is resolved by ignoring the `GOTO` instructions during the debugging process.

## Automation tests
 - Unit tests 
   > Added

### CODE REVIEW

File: bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BVM.java [hunk 1]
```diff
@@ -3090,6 +3090,11 @@ private static boolean debug(Strand ctx) {
             return false;
         }
 
+        int opcode = ctx.currentFrame.code[ctx.currentFrame.ip].getOpcode();
```
Comment (lankavitharana) [inline bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BVM.java]: shall we move this to the method?

Comment (pranavan15) [reply bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/BVM.java]: +1. Commit 726d308 addresses this.

File: tests/ballerina-unit-test/src/test/resources/test-src/debugger/test_nested_if.bal [hunk 2]
```diff
@@ -0,0 +1,13 @@
+public function main(string... args) {
```
Comment (hemikak) [inline tests/ballerina-unit-test/src/test/resources/test-src/debugger/test_nested_if.bal]: Need copyright header

Comment (pranavan15) [reply tests/ballerina-unit-test/src/test/resources/test-src/debugger/test_nested_if.bal]: Added with commit 9ec303d.

**Comment by lankavitharana** [comment]
I guess you have rebased this with the master(hence the force push message) I think it would be better to merge rather than rebasing in future.

**Pull request merged**